examples/README.md

```diff
@@ -88,3 +88,75 @@ cosign attest --yes --type spdxjson \
   --predicate sboms/example.spdx.json \
   "${IMAGE}"
 ```
+
+### signed-by-aws-kms-key
+
+Source:  [policies/signed-by-aws-kms.yaml](./policies/signed-by-aws-kms.yaml)
+
+Asserts that images have been signed by a specific AWS KMS key.
```

Comment:
Nit: below here (and the other one), can we add

```
POLICY="policies/signed-by-aws-kms-key.yaml"
```

for copy-paste goodness and to have parity with the others